### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by YouKnowBlom**
Currently eslint is disabled for most files. Plan is to gradually fix them instead of shoving it all into this PR

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    root: true,
+    env: {
+        node: true,
+        es6: true
+    },
+    extends: [
+        'eslint:recommended'
+    ],
+    parserOptions: {
+        ecmaVersion: 2015,
```
Comment (dkanada) [inline .eslintrc.js:11]: might as well use 2020

Comment (YouKnowBlom) [reply .eslintrc.js:11]: Sounds good. I wasn't sure if there was much support for es2020 yet but it seems like chrome supports a lot of the new stuff so we should be good👍

**Comment by YouKnowBlom** [comment]
> Are these files transpiled for older browsers? I was planning on setting up a nightly but that would be a prerequisite since I'm sure a lot of clients are extremely old.

They are currently not transpiled. Afaik all cast devices run a recent version of chrome so I don't think that should be an issue, unless the user has somehow disabled automatic updates.

File: .eslintrc.js (hunk 1 above)
Comment (MrTimscampi) [reply .eslintrc.js]: You can probably throw Babel in there if you need another version in the browser :)

**Pull request merged**